pandas/core/generic.py

```diff
@@ -1930,25 +1930,52 @@ def to_pickle(self, path, compression='infer',
 
     def to_clipboard(self, excel=True, sep=None, **kwargs):
         """
-        Attempt to write text representation of object to the system clipboard
+        Copy object to the system clipboard.
+
+        Write a text representation of object to the system clipboard.
         This can be pasted into Excel, for example.
 
         Parameters
         ----------
-        excel : boolean, defaults to True
-                if True, use the provided separator, writing in a csv
-                format for allowing easy pasting into excel.
-                if False, write a string representation of the object
-                to the clipboard
-        sep : optional, defaults to tab
-        other keywords are passed to to_csv
+        excel : bool
+            Default setting for this argument is True.
+            If True, use the provided separator, writing in a csv format for
+            allowing easy pasting into excel.
+            If False, write a string representation of the object to the
+            clipboard.
+        sep : str, default tab
+            Field delimiter.
+        kwargs : optional
+            These parameters will be passed to either
+            :py:meth:`pandas.DataFrame.to_csv`
+            or :py:meth:`pandas.Series.to_csv` methods depending on the
+            Object type.
```

Comment:
Since they actually have the same optional keyword, I would just refer to DataFrame.to_csv to keep it a bit shorter